test/test_other.py

```diff
@@ -5198,15 +5198,17 @@ def test_bad_lookup(self):
     'full_only': [{'EMCC_FORCE_STDLIBS': 'libc,libc++abi,libc++,libmalloc', 'EMCC_ONLY_FORCED_STDLIBS': '1'}, False],
   })
   def test_only_force_stdlibs(self, env, fail):
+    cmd = [EMXX, test_file('hello_libcxx.cpp'), '-sWARN_ON_UNDEFINED_SYMBOLS=0']
```

Comment:
Does setting `WARN_ON_UNDEFINED_SYMBOLS=0` matter here? IIUC with it set to 0 we still error as `ERROR_ON_*` is set to 1 anyhow. So this `-s` here has no effect. In fact, looking in the test suite, I don't see a proper test for this flag at all - am I missing something?

Reply:
There `-sWARN_ON_UNDEFINED_SYMBOLS=0` flag here exist in both before and after this patch.  I just moved it.

`-sWARN_ON_UNDEFINED_SYMBOLS=0` does imply `-sERROR_ON_UNDEFINED_SYMBOLS=0`.

But looking at this change again, it seems like we shouldn't need to update this test..  taking another look now.